Contents/Code/siteData18Movies.py

```diff
@@ -123,7 +124,7 @@ def update(metadata, siteNum, movieGenres, movieActors):
 
     # Summary
     summary = detailsPageElements.xpath('//div[@class="gen12"]/p[contains(., "Description")]')[0].text_content().split(':', 1)[1].strip()
-    if len(summary) > 1:
```

Comment:
I did this because one of the summaries I found was a 0. I didn't feel like checking all the instances of a bad summary so I made it a length of 2 min.

Reply:
Returned